pandas/tests/io/test_parquet.py

```diff
@@ -478,6 +480,25 @@ def test_s3_roundtrip(self, df_compat, s3_resource, pa):
         check_round_trip(df_compat, pa,
                          path='s3://pandas-test/pyarrow.parquet')
 
+    def test_partition_cols_supported(self, pa_ge_070, df_full):
+        partition_cols = ['bool', 'int']
+        df = df_full
+        path = tempfile.mkdtemp()
+        df.to_parquet(path, partition_cols=partition_cols,
+                      compression=None)
```

Comment:
is there a reason you are also not reading this back with ``pd.read_parquet``?

Reply:
The objective of this test was to to ensure the file is partitioned. Hence covered that alone in this test. I am not clear on how to get the number of partitions in the file using read_parquet